server/lib/configuration.js

```diff
@@ -149,6 +149,32 @@ var conf = module.exports = convict({
     doc: 'Google Analytics id',
     format: String
   },
+  hpkp_config: {
+    enabled: {
+      default: true,
```

Comment:
I have tested using local-dev and works as expected, will deploy a fxa-dev stack and verify no breakage.